inst/sympref.m

```diff
@@ -306,6 +306,9 @@
             msg = 'Choosing the default [autodetect] communication mechanism';
           case 'pytave'
             msg = 'Forcing the PyTave communication mechanism';
+            pyexec(strjoin({'import sympy as sp',
+                            'from sympy import __version__ as spver', 
+                            'from sympy import *'}, newl));
```

Comment:
this is not the right place, the first time `python_ipc_pytave` is called, use a `persistent` variable to keep track of its the first time.